actors/builtin/power/power_actor.go

```diff
@@ -37,15 +37,12 @@ func (a Actor) Exports() []interface{} {
 		2:                         a.CreateMiner,
 		3:                         a.DeleteMiner,
 		4:                         a.OnSectorProveCommit,
-		5:                         a.OnSectorTerminate,
-		6:                         a.OnFaultBegin,
-		7:                         a.OnFaultEnd,
-		8:                         a.OnSectorModifyWeightDesc,
-		9:                         a.EnrollCronEvent,
-		10:                        a.OnEpochTickEnd,
-		11:                        a.UpdatePledgeTotal,
-		12:                        a.OnConsensusFault,
-		13:                        a.SubmitPoRepForBulkVerify,
+		5:                         a.UpdateClaimedPower,
```

Comment:
It's a little disturbing that this change affects no tests.

Reply:
Right. Because The `OnSectorProveCommit` is still a distinct method for the power actor, all the test scenarios committing new sectors weren't touched. We need to cover the lifecycle events of the removed methods (faults, recovery, termination).